pages/how-to/upgrade/plural-cli.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Upgrading the Plural CLI
```

Comment:
don't think we need these docs or at least not here if it's going to rabbithole users